tests/lua-empty-test/project/proj.android/app/build.gradle

```diff
@@ -114,11 +114,19 @@ def getCocosCommandPath() {
         return 'cocos.bat'
     }
     else {
+
+        def env_console_path = System.getenv("COCOS_CONSOLE_ROOT")
+
+        if (env_console_path != null && !env_console_path.trim().isEmpty()) {
+            return new File(env_console_path.trim() + '/cocos').absolutePath
+        }
+
         // on unix like system, can not get environments variables easily
         // so run a shell script to get environment variable sets by cocos2d-x setup.py
         new ByteArrayOutputStream().withStream { os ->
             def result = exec {
-                executable = project.file('get_environment.sh')
+                executable = "/bin/bash"
+                args = [ "-i", project.file('get_environment.sh')]
```

Comment:
https://linux.die.net/man/1/bash

> When bash is invoked as an interactive login shell, or as a non-interactive shell with the  --login  option,
       it  first  reads  and executes commands from the file /etc/profile, if that file exists.  After reading that
       file, it looks for ~/.bash_profile, ~/.bash_login, and ~/.profile, in that order,  and  reads  and  executes
       commands  from the first one that exists and is readable.  The --noprofile option may be used when the shell
       is started to inhibit this behavior.